### DISCUSSION TITLE
Fancybox 4 - Custom CSS and Options

### DISCUSSION THREAD

**Discussion by marianohsn** [fancyapps/ui#355] [General]
Longtime user of Fancybox, trying to move from v3 to v4 and I just can't figure out a couple basic things:

It seems like Fancybox is setup for a carousel by default, hence why I'm seeing the following:

- **How do I disable the default 'grab and slide' movement and make Fancybox zoom out/close when I click/tap on the opened image?** Zoomed/opened image doesn't close when I click on it, instead, the cursor changes to an open hand, and the image slides horizontally and bounces back when I drag it. It does close when I drag it vertically but that's something I'd like to disable as well.

And lastly:
- **How do I change the overlay color?**

Thanks

**Comment by b-s-j** [comment]
sir do you know how to maintain the fancybox 4 on its state even if I click the backdrop it will not close ?
